Mathlib/Analysis/Normed/Field/Basic.lean

```diff
@@ -454,6 +454,19 @@ lemma nnnorm_sub_mul_le (ha : ‖a‖₊ ≤ 1) : ‖c - a * b‖₊ ≤ ‖c -
 chord length is a metric on the unit complex numbers. -/
 lemma nnnorm_sub_mul_le' (hb : ‖b‖₊ ≤ 1) : ‖c - a * b‖₊ ≤ ‖1 - a‖₊ + ‖c - b‖₊ := norm_sub_mul_le' hb
 
+lemma norm_commutator_sub_one_le (a b : αˣ) :
```

Comment:
perhaps
```suggestion
lemma norm_commutator_units_sub_one_le (a b : αˣ) :
```
to leave room for a `DivisionRing` version as well.

... can you add that DivisionRing version? is the bound tight there?

Reply:
No the bound is not tight because of the `‖(a - 1) * (b - 1) - (b - 1) * (a - 1)‖ ≤ ‖(a - 1) * (b - 1)‖ + ‖(b - 1) * (a - 1)‖` step. It is tight in a normed field. I guess I will add both versions

Reply:
In a normed field the statement is pretty silly because the LHS is 0

Reply:
Wait no of course the normed field version is not tight. The LHS is zero!